### ISSUE TITLE
Always equalize the area of slits of upper right and lower right.

### ISSUE THREAD

**Issue by satakagi** [cytan299/tribahtinov#2]
I have created a function to calculate the boundary line which makes the area of upper right slits equal to the area of lower right slits. This page determines its boundary by this function.
[http://svg2.mbsrv.net/astro/Tri-Bahtinov_r2a.html](http://svg2.mbsrv.net/astro/Tri-Bahtinov_r2a.html)

I also placed a checkbox for the function to build mount holes and the function to equalize the width of boundary stems and slits. I would like to release it after allowing the original boundary calculation logic to be selected. Are these features meaningful?

**Comment by cytan299** [comment]
Hi Satoru,   Those are great ideas! One option, I think might be useful is for the user to select the width of the stem rather than to making them equal. You can have equal slit width and stem width as the default when this option is selected. i.e.         slit width to stem width ratio = 1:1
as defaultand then the user can enter for example:       slit width to stem width ratio = 1.5:1
What do you think?
cytan

**Comment by satakagi** [comment]
Hi, cytan.
Thanks for your comment. I just completed coding and simple testing!
[http://svg2.mbsrv.net/astro/Tri-Bahtinov_r2a.html](http://svg2.mbsrv.net/astro/Tri-Bahtinov_r2a.html)

**Comment by cytan299** [comment]
Hi Satoru,   Excellent programming! You are a much better programmer than I am. Looks really good. You should really announce what you did on the forums.  I have no doubt that your work will be a fantastic contribution to the astrophotography community.
cytan

**Comment by satakagi** [comment]
Hi, cytan, Thank you for your reply.
I happened to develop WWW technology as a profession and had a hobby of astrophotography, so I just could port your cool ideas and logic as a web application. And I am also pleased that I made a little improvement contribution. Of course, the same open source license declaration is listed in the source code of webApps I ported.

By the way, since I'd like to introduce Tri-Bahtinov mask also to Japanese people, I have translated readme and wiki of this repository which describes its mechanism and usage. I want to issue a pull request to add these translations.
